nautobot/docs/installation/region-and-site-data-migration-guide.md

```diff
@@ -0,0 +1,405 @@
+    "parent__name__in": ["United States", "Greece", "England"],
+    "parent__slug": "united-states",
+    "parent__slug__in": ["united-states", "greece", "england"],
+    "parent__id": "9e0e9fc2-0326-40db-b3a3-3afe2207c1ad",
+    "parent__id__in": ["9e0e9fc2-0326-40db-b3a3-3afe2207c1ad", "9e0e9fc2-0326-40db-2222-222222222222"],
+    "parent__parent__name": "North America",
+    "parent__parent__name__in": ["North America", "Europe"],
+    "parent__parent__slug": "north-america",
+    "parent__parent__slug__in": ["north-america", "europe"],
+    "parent__parent__id": "9mkl123094fc2-0326-40db-b3a3-3afe2207c1ad",
+    "parent__parent__id__in": ["9mkl123094fc2-0326-40db-b3a3-3afe2207c1ad", "82132yhks-875njjni121-0091213"],
+}
+```
+
+### Other Data Model Specific ObjectPermission e.g. Interface
+
+The old `constraints` field for a `Site`/`Region` related data model's (e.g. `Interface`) `ObjectPermission` instance might look like this:
+
+```json
+{
+    "device__site__name": "AMS01",
+    "device__site__name__in": ["AMS01", "ATL01", "ETX02"],
+    "device__site__slug": "ams01",
+    "device__site__slug__in": ["ams01", "atl01", "etx02"],
+    "device__site__id": "0ab47314-2944-45f6-b964-9e009fc48ce0",
+    "device__site__id__in": ["0ab47314-2944-45f6-b964-9e009fc48ce0", "b09545d4-6e2b-471e-8f07-27f25ca308f5"],
+    "device__site__region__name": "United States",
+    "device__site__region__name__in": ["United States", "United Kingdom", "Greece"],
+    "device__site__region__slug": "united-states",
+    "device__site__region__slug__in": ["united-states", "united-kingdom", "greece"],
+    "device__site__region__id": "0ab47314-123144-2132f6-b964-9ekmwkq12c48ce0",
+    "device__site__region__id__in": ["0ab47314-123144-2132f6-b964-9ekmwkq12c48ce0", "kml21345d4-6e123-471e-8as1231-27f25lmsahca308f5"],
+    "device__site__region__parent__name": "North America",
+    "device__site__region__parent__name__in": ["North America", "Europe", "South America"],
+    "device__site__region__parent__slug": "north-america",
+    "device__site__region__parent__slug__in": ["north-america", "europe", "south-america"],
+    "device__site__region__parent__id": "0kmskqjneq14-945840-2132f6-b964-9ekmwkq12c48ce0",
+    "device__site__region__parent__id__in": ["0kmskqjneq14-945840-2132f6-b964-9ekmwkq12c48ce0", "qwe0kmjn1lohg-12olm-kmlie-8as1231-km2l1321isdan234f5"],
+}
+```
+
+To modify the data correctly, we need to replace query filter strings to those of model class `Location`:
+    1. Replace all occurrences of "site" with "location" in the **Key** portion (before ":") of the data **not Value** (after ":").
+    2. Replace all occurrences of "region" with "parent" in the **Key** portion (before ":") of the data **not Value** (after ":").
+    3. Since we are keeping the uuids of the newly created "Site"/"Region" type locations the same from those of the old `Site`/`Region` instances, we do not need to change the uuid values in "..__id" and "..__id__in" Keys.
```

Comment:
```suggestion
    3. Since Nautobot carries over the UUIDs of the old `Site`/`Region` instances when creating the new "Site"/"Region" type `Location` instances, we do not need to change the UUID values in `...__id` and `...__id__in` Keys.
```